src/conditions/conditions.component.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  DataTable,
+  DataTableSkeleton,
+  Layer,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+  Pagination,
+} from '@carbon/react';
+import { ErrorState, formatDate, parseDate, usePagination } from '@openmrs/esm-framework';
+import { CardHeader, EmptyState } from '@openmrs/esm-patient-common-lib';
```

Comment:
Seems preferable that we move these from common-lib to the styleguide. Adding a dependency on common-lib here is ideally not something we would do.

Reply:
We should ticket this for later.